rule-engine/rule-engine-components/src/main/java/org/thingsboard/rule/engine/action/TbCreateAlarmNode.java

```diff
@@ -67,10 +69,19 @@ protected TbCreateAlarmNodeConfiguration loadAlarmNodeConfig(TbNodeConfiguration
     @Override
     protected ListenableFuture<TbAlarmResult> processAlarm(TbContext ctx, TbMsg msg) {
         String alarmType;
+        String alarmSeverity;
```

Comment:
Store this as an enumeration. not a String.

Reply:
@ashvayka Refactored